src/schemas/21-686C/schema.js

```diff
@@ -285,129 +285,228 @@ const commonAddressFields = {
-    },
-    postalCode,
-    country: {
-      type: 'string',
-      maxLength: 50,
-      default: countryUSA
-    }
-  },
-  additionalProperties: false
-}
-
-const militaryAddress = {
-  required: [...commonAddressFields.required, 'postOffice', 'postalType', 'postalCode'],
-  properties: {
-    ...commonAddressFields.properties, 
-    addressType: {
-      type: 'string',
-      enum: ['MILITARY'],
-      default: 'MILITARY'  
-    },
-    postOffice: {
-      type: 'string',
-      'enum': [
-        'APO',
-        'DPO',
-        'FPO'
-      ],
-      enumNames: [
-        'Ashore Post Office',
-        'Diplomatic Post Office',
-        'Fleet Post Office'
-      ]
-      
-    },
-    postalType: {
-      type: 'string',
-      'enum': [
-        'AA',
-        'AE',
-        'AP'
-      ],
-      enumNames: [
-        'Armed Forces Americas (except Canada)',
-        'Armed Forces Europe, Middle East, Canada & Africa',
-        'Armed Forces Pacific'
-      ]
-    },
-    postalCode
-  },
-  additionalProperties: false
-}
-
-
-const internationalAddressDropDown = {
-  required: [...commonAddressFields.required],
+const commonMarriageDef = {
+  required: ['dateOfMarriage', 'locationOfMarriage', 'spouseFullName'],
   properties: {
-    ...commonAddressFields.properties,
-    addressType: {
-      type: 'string',
-      enum: ['INTERNATIONAL'],
-      default: 'INTERNATIONAL'
+    dateOfMarriage: schemaHelpers.getDefinition('date'),
+    locationOfMarriage: {
+      $ref: '#/definitions/location'
     },
-    country: {
-      type: 'string',
-      'enum': countriesForeign
-    }
-  },
-  additionalProperties: false
-}
-
-const internationalAddressText = {
-  required: [...commonAddressFields.required, 'countryText'],
-  properties: {
-    ...commonAddressFields.properties,
-    addressType: {
-      type: 'string',
-      enum: ['INTERNATIONAL'],
-      default: 'INTERNATIONAL'
-    },
-    country: {
-      type: 'string',
-      'enum': [countryNotInList],
-      default: countryNotInList
-    },
-    countryText: {
-      type: 'string',
-      maxLength: 50,
-      minLength: 1,
-      pattern: textRegex
-    }
-  },
-  additionalProperties: false
+    spouseFullName: schemaHelpers.getDefinition('fullName')
+  }
 }
 
-
-const addressDefs = [domesticAddress, militaryAddress, internationalAddressDropDown, internationalAddressText];
+// const addressDefs = [domesticAddress, militaryAddress, internationalAddressDropDown, internationalAddressText];
+const addressDefs = [
+  { $ref: '#/definitions/domesticAddress' },
+  { $ref: '#/definitions/militaryAddress' },
+  { $ref: '#/definitions/internationalAddressDropDown' },
+  { $ref: '#/definitions/internationalAddressText' }
+]
 
 let schema = {
   $schema: 'http://json-schema.org/draft-04/schema#',
   title: 'DECLARATION OF STATUS OF DEPENDENTS',
   type: 'object',
   additionalProperties: false,
-  definitions,
+  definitions: _.merge(_.pick(definitions, 'fullName'),
```

Comment:
`_.pick(definitions, ['fullName'])` is already being called above, how about calling `_.merge(definitions, ...)` here?